pallets/process-validation/src/restrictions.rs

```diff
@@ -271,4 +301,76 @@ mod tests {
         );
         assert!(!result);
     }
+
+    #[test]
+    fn fixed_metadata_value_outputs_total() {
```

Comment:
We probably need some more test cases here. For example:

* wrong value under correct key
* correct value under incorrect key
* correct value on correct key on wrong input

Reply:
Done